serving/samples/helloworld-java-spark/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use the official maven/Java 8 image to create a build artifact.
+# https://hub.docker.com/_/maven
+FROM maven:3.5-jdk-8-alpine as builder
+
+# Copy local code to the container image.
+WORKDIR /app
+COPY pom.xml .
+COPY src ./src
+
+# Build a release artifact.
+RUN mvn package -DskipTests
+
+# Use the Official OpenJDK image for a lean production stage of our multi-stage build.
+# https://hub.docker.com/_/openjdk
+# https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
+FROM openjdk:8-jre-alpine
+
+# Copy the jar to the production image from the builder stage.
+COPY --from=builder /app/target/helloworld-0.0.1-SNAPSHOT-jar-with-dependencies.jar /helloworld.jar
+
+# Service must listen to $PORT environment variable.
+# This default value facilitates local development.
+ENV PORT 8080
+
+# Run the web service on container startup.
+CMD ["java","-Dserver.port=${PORT}","-jar","/helloworld.jar"]
```

Comment:
Does this respect the PORT environment variable changing along the lines of `docker run --rm -e PORT=9090 [...]`? Did a local experiment and it doesn't seem to work.

helloworld-python uses: `CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 app:app`

Reply:
@grayside I need to test this out.